.github/ISSUE_TEMPLATE/presentation.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Presentation
+about: Have something you want to share with the group? Propose a presentation for the SIG-Security weekly meetings. Presentations can include project sharing, use-case sharings, and any cloud native and security related topics.
```

Comment:
maybe add after "Have something you want to share with the group?"

"or a topic you would like to hear presented? or someone you would like to invite to speak about a relevant topic?"

Reply:
Yea i wasn't sure whether wanting to hear something should fall under suggestion? I think either is fine though, and we can triage as necessary.

Reply:
Updated!